HandheldCompanion/ViewModels/GameViewModel.cs

```diff
@@ -0,0 +1,297 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using craftersmine.SteamGridDBNet;
+using GameLib;
+using GameLib.Core;
+using GameLib.Plugin.RiotGames.Model;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using static Microsoft.WindowsAPICodePack.Shell.PropertySystem.SystemProperties.System;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace HandheldCompanion.ViewModels;
+
+public partial class GameViewModel : ViewModelBase
+{
+    const string CrossImagePath = "/Resources/GameLib/cross-color.png";
+    const string CheckImagePath = "/Resources/GameLib/check-color.png";
+
+    private readonly LauncherManager _launcherManager;
+
+    [ObservableProperty]
+    private ObservableCollection<GameExtended> _games = default!;
+
+    [ObservableProperty]
+    private GameExtended? _selectedGame;
+
+    [ObservableProperty]
+    private bool _isLoading;
+
+    [ObservableProperty]
+    private bool _noGameFound = false;
+
+    [ObservableProperty]
+    private string _isRunningLogo = CrossImagePath;
+
+    [ObservableProperty]
+    private string? _launcherName;
+
+    [ObservableProperty]
+    private string? aPIKey = "3211a8c119e2d7e013e69e80d9bc460a";
+
+    private static SteamGridDb _steamGridDb { get; set; }
+
+    public GameViewModel(LauncherManager launcherManager)
+    {
+        _launcherManager = launcherManager;
+        _steamGridDb = new SteamGridDb(aPIKey);
+        LoadData();
+    }
+
+    private async void LoadData()
```

Comment:
Never do async void, unless you know what you are doing This should be:
`private async Task LoadData()`